doc/usage/extensions/autosummary.rst

```diff
@@ -54,6 +59,27 @@ The :mod:`sphinx.ext.autosummary` extension does this in two parts:
 
        .. currentmodule:: sphinx.ext.autosummary
 
+   When :confval:`autosummary_recursion_limit != 0`, the :rst:dir:`autosummary`
+   directive recursively generates stub ``.rst`` files for packages.
+
+   For example, ::
```

Comment:
Is this example related with above sentence? It seems `autosummary_recursion_limit` is not set on this document.

Reply:
paragraph removed altogether